connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1957,6 +1958,29 @@ object functions {
    */
   def try_sum(e: Column): Column = Column.fn("try_sum", e)
 
+  /**
+   * Casts the column to a different data type and the result is null on failure.
+   *
+   * @group normal_funcs
+   * @since 4.0.0
+   */
+  def try_cast(e: Column, dataType: DataType): Column = Column { builder =>
```

Comment:
I think we should have it as `Column` method same as `cast`?

Reply:
sure, let me change it to `Column.try_cast`